h2o-algos/src/main/java/hex/schemas/GLMV3.java

```diff
@@ -310,13 +351,46 @@ public static final class GLMParametersV3 extends ModelParametersSchemaV3<GLMPar
             level = API.Level.secondary, direction = API.Direction.INOUT)
     public double tweedie_epsilon;
     
-    @API(help = "Control the maximum number of iterations in the dispersion parameter estimation loop using maximum likelihood.", level = API.Level.secondary, direction = API.Direction.INOUT)
+    @API(help = "Control the maximum number of iterations in the dispersion parameter estimation loop using maximum" +
+            " likelihood.", level = API.Level.secondary, direction = API.Direction.INOUT)
     public int max_iterations_dispersion;
 
     @API(help="If set to true, will generate scoring history for GLM.  This may significantly slow down the algo.", 
             level = Level.secondary, direction = Direction.INPUT)
     public boolean generate_scoring_history;  // if enabled, will generate scoring history for iterations specified in
                                               // scoring_iteration_interval and score_every_iteration
+    
+    @API(help="If true, will initialize coefficients with values derived from GLM runs without linear constraints.  " +
+            "Only available for linear constraints.  Default to false.", level = API.Level.secondary, 
+            direction = API.Direction.INOUT, gridable = true)
+    public boolean init_optimal_glm;
+
+    @API(help="If true, will keep the beta constraints and linear constraints separate.  After new coefficients are" +
+            "found, first beta constraints will be applied followed by the application of linear constraints.  Note " +
+            "that the beta constraints in this case will not be part of the objective function.  If false, will" +
+            " combine the beta and linear constraints.  Default to false.", level = API.Level.secondary,
+            direction = API.Direction.INOUT, gridable = true)
+    public boolean separate_linear_beta;
+
+    @API(help="For constrained GLM only.  It affects the setting of eta_k+1=eta_0/power(ck+1, alpha).", 
```

Comment:
I miss the default info. Also for tau, alpha, beta and c0.

Reply:
I added to the front part of GLMV3.java: 
![image](https://github.com/h2oai/h2o-3/assets/7231712/4b321d26-b111-4440-a8a7-e6828b785ca6)

Reply:
Ok. I think it would be nice to have this information also in API help, but It is not critical.